src/jvm/main/org/jetbrains/kotlinx/lincheck/transformation/transformers/MethodCallTransformer.kt

```diff
@@ -173,39 +118,32 @@ internal class MethodCallTransformer(
         }
     }
 
-    private fun isIgnoredMethod(owner: String, methodName: String) =
-        owner.startsWith("sun/nio/ch/lincheck/") ||
-        owner.startsWith("org/jetbrains/kotlinx/lincheck/") ||
-        owner == "kotlin/jvm/internal/Intrinsics" ||
-        owner == "java/util/Objects" ||
-        owner == "java/lang/String" ||
-        owner == "java/lang/Boolean" ||
-        owner == "java/lang/Long" ||
-        owner == "java/lang/Integer" ||
-        owner == "java/lang/Short" ||
-        owner == "java/lang/Byte" ||
-        owner == "java/lang/Double" ||
-        owner == "java/lang/Float" ||
-        owner == "java/util/Locale" ||
-        owner == "org/slf4j/helpers/Util" ||
-        owner == "java/util/Properties"
-
-    private fun isAtomicClass(className: String) =
-        className == "java/util/concurrent/atomic/AtomicInteger" ||
-        className == "java/util/concurrent/atomic/AtomicLong" ||
-        className == "java/util/concurrent/atomic/AtomicBoolean" ||
-        className == "java/util/concurrent/atomic/AtomicReference"
+    private fun isIgnoredMethod(className: String, methodName: String) =
+        className.startsWith("sun/nio/ch/lincheck/") ||
+        className.startsWith("org/jetbrains/kotlinx/lincheck/") ||
+        className == "kotlin/jvm/internal/Intrinsics" ||
+        className == "java/util/Objects" ||
+        className == "java/lang/String" ||
+        className == "java/lang/Boolean" ||
+        className == "java/lang/Long" ||
+        className == "java/lang/Integer" ||
+        className == "java/lang/Short" ||
+        className == "java/lang/Byte" ||
+        className == "java/lang/Double" ||
+        className == "java/lang/Float" ||
+        className == "java/util/Locale" ||
+        className == "org/slf4j/helpers/Util" ||
+        className == "java/util/Properties"
 
-    private fun isAtomicArrayClass(className: String) =
-        className == "java/util/concurrent/atomic/AtomicIntegerArray" ||
-        className == "java/util/concurrent/atomic/AtomicLongArray" ||
-        className == "java/util/concurrent/atomic/AtomicReferenceArray"
+    private fun isExceptionThrowingMethod(className: String, methodName: String) =
```

Comment:
This is a performance optimization. Do we need it?

Reply:
I am not sure. I added it so the code functionally remains the same as before.

Reply:
Could you investigate, please? The shorter the code the better.

Reply:
I checked on my machine. I do not observe any difference, so I removed special handling of atomic methods.